docs/DeployingOnCentOS7.md

```diff
@@ -71,74 +105,66 @@ Example `firewalld` config
 
-    $ git clone git@github.com:mesosphere/net-modules.git
-    $ cd net-modules
-    $ make builder-rpm
-    $ ls packages/rpms/RPMS/x86_64
+Also note, you will need to fill in the IP address of the host that is running ZooKeeper.
 
-The files you see in the `packages/rpms/RPMS/x86_64` directory are all the packages you need to install Mesos, while additionally including the compiled net-modules library. Transfer these files to each host in your cluster.
+    MESOS_QUORUM=1
+    MESOS_WORK_DIR=/var/lib/mesos
+    MESOS_ZK=zk://<IP of host with zookeeper running>:2181/mesos
 
-## Install the packages
+If ZooKeeper is not running on your master, you will need to go into `/etc/mesos/zk` and change the line to include the IP address of the host with ZooKeeper running.
 
-If you followed the guide correctly so far, you should have the Mesos rpm packages on each of your master and agent hosts.
+## Run Mesos Master
 
-Before installing these packages, you must have the Extra Packages for Enterprise Linux (or EPEL) packages installed.
+Run the mesos-master process on your master host.
 
-    $ sudo yum -y install epel-release
+    $ sudo systemctl enable mesos-master.service
+    $ sudo systemctl start mesos-master.service
 
-Now your server should be ready to install the Mesos packages.
+# Configure Agent
+## Configure your firewall
 
-    $ sudo yum -y install *.rpm
+You will either need to configure the firewalls on each node in your cluster (recommended) to allow access to the cluster services or disable it completely.  Included in this section is configuration examples for `firewalld`.  If you use a different firewall, check your documentation for how to open the listed ports.
 
-# Run the Backend Services
+Agent (compute) nodes require
 
-You will need to run one instance of both, etcd and ZooKeeper. Most Mesos cluster will run the backend services in a cluster external to their core Mesos cluster. For simplicity's sake, we will be launching ZooKeeper and etcd on the master host.
+| Service Name | Port/protocol     |
+|--------------|-------------------|
+| BIRD (BGP)   | 179/tcp           |
+| mesos-agent  | 5051/tcp          |
 
-Both services will be run in Docker containers.
+Example `firewalld` config
 
-Also, keep note of the IP address of the host you deploy on because we will need this later when configuring our Mesos masters and agents.
+    $ sudo firewall-cmd --zone=public --add-port=179/tcp --permanent
+    $ sudo firewall-cmd --zone=public --add-port=5051/tcp --permanent
+    $ sudo systemctl restart firewalld
 
-## ZooKeeper
+## Install Docker
 
-    $ sudo docker pull jplock/zookeeper:3.4.5
-    $ sudo docker run --detach --name zookeeper -p 2181:2181 jplock/zookeeper:3.4.5
+    $ sudo yum -y install docker docker-selinux
+    $ sudo systemctl enable docker.service
+    $ sudo systemctl start docker.service
 
-## etcd
+## Verify Docker installation
```

Comment:
It feels weird that this is only in the agent when we tell the user to run etcd and zookeeper in a docker container on the master. Should the Docker section be in the `Prepare external services section`?

No need to make this change, I can once I pull this into my branch. Just want your input